### ISSUE TITLE
Podman Desktop is smaller on Linux

### ISSUE THREAD

**Issue by gastoner** [containers/podman-desktop#8550]
### Bug description

Compared to Windows, WSL Linux version is noticeably smaller 

Left: Linux Right: W
![image](https://github.com/user-attachments/assets/e2f7a2f8-880d-46a6-8cd2-bd93484d98e4)

### Operating system

Windows 11/ WSL Linux

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

**Comment by axel7083** [comment]
![image](https://github.com/user-attachments/assets/9bcac3e5-1b0d-431c-ba05-e07d4ed64d88)

On the left I have a windows remote desktop session, and on the right the podman-desktop from my fedora 40. I don't have the impression that there are much difference

**Comment by jeffmaury** [comment]
On the same topic; on Linux some fonts are hard to read:

![image](https://github.com/user-attachments/assets/cdbfd1f6-21e5-4d66-afbb-2d8d75a86b7d)

**Comment by gastoner** [comment]
> ![image](https://private-user-images.githubusercontent.com/42176370/361880841-9bcac3e5-1b0d-431c-ba05-e07d4ed64d88.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MjI0MzUsIm5iZiI6MTcyNDgyMjEzNSwicGF0aCI6Ii80MjE3NjM3MC8zNjE4ODA4NDEtOWJjYWMzZTUtMWIwZC00MzFjLWJhMDUtZTA3ZDRlZDY0ZDg4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDA1MTUzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0ODM4YTBlNzMyMGE1NDQ4MmI1YmEwMzE1OTlkOGI2NGY1ZmQ5YzA0NGQyNDZiYWQwNDEzZjdmNWE2N2NkZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Hwh3PSVbnjFs97zihD4IhIIr9gLqUAPkW4R8JEFbs7M)
> 
> On the left I have a windows remote desktop session, and on the right the podman-desktop from my fedora 40. I don't have the impression that there are much difference

This might be just WSL Linux thing then

**Comment by cdrage** [comment]
> ![image](https://private-user-images.githubusercontent.com/42176370/361880841-9bcac3e5-1b0d-431c-ba05-e07d4ed64d88.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NjU1MTUsIm5iZiI6MTcyNTU2NTIxNSwicGF0aCI6Ii80MjE3NjM3MC8zNjE4ODA4NDEtOWJjYWMzZTUtMWIwZC00MzFjLWJhMDUtZTA3ZDRlZDY0ZDg4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDE5NDAxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY5NGEzOTBhMWM4MDMxZTVlNTkxZjBkMWMxNzgzNWFlMGU3NTZkMGYyOGJmZWFkMWMyMmUwZDJiYWZiYWZlNjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SGh955IvIgJiVmJBXanOKTJhBBzSrN2xSKbH_6dnuwY)
> 
> On the left I have a windows remote desktop session, and on the right the podman-desktop from my fedora 40. I don't have the impression that there are much difference

I got the exact same as @axel7083 trying to reproduce this but was unable to.



> On the same topic; on Linux some fonts are hard to read:
> 
> ![image](https://private-user-images.githubusercontent.com/695993/361898280-cdbfd1f6-21e5-4d66-afbb-2d8d75a86b7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NjU1MTUsIm5iZiI6MTcyNTU2NTIxNSwicGF0aCI6Ii82OTU5OTMvMzYxODk4MjgwLWNkYmZkMWY2LTIxZTUtNGQ2Ni1hZmJiLTJkOGQ3NWE4NmI3ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQxOTQwMTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTNjMzNiNTc2ODE1YTg4MWZkOGZhMzgzMWUzNDc3MjE5MDcxY2EyYWY1ZDBhYTU2MDM0OTdhNzNlMDk0Zjc0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.eiwD_Qheh-KmyPHARsyJCLRR1roMO9iiZqg9sJpuEcM)

This has been fixed in the main branch I believe as I am not getting this with the dark / light theme set.

Tried the above fixes on Windows 10 as well as Fedora 40.

**Comment by gastoner** [comment]
@cdrage This is probably just WSL fedora thing, I don't have laptop with just fedora so I use WSL where I have Fedora

**Comment by cdrage** [comment]
After some investigation the fonts are noticeable smaller in WSL which is intentional with the way that WSL runs underlying applications.

This can be resolved using the zoom functionality added by @benoitf 

I do not believe we can fix this as I tried opening up another electron app (lens) and noticed the font was also smaller too, so it's valid to say it's intentional on all applications.

Closing! Let me know if we should re-open.
